src/features/useFetch.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+
+
+function useFetch(urls) {
```

Comment:
if you intend for this to be a custom Hook, then it this file is better placed under a folder you can call `hooks` not under features
Always think "**separation of concern**"

Reply:
Done